pkg/dataquality/warnings.go

```diff
@@ -37,3 +37,17 @@ func WarnDisconnectedTrace(tenant string, phase string) {
 func WarnRootlessTrace(tenant string, phase string) {
 	metric.WithLabelValues(tenant, reasonRootlessTrace+phase).Inc()
 }
+
+var MetricSpanInFuture = promauto.NewHistogramVec(prometheus.HistogramOpts{
+	Namespace: "tempo",
+	Name:      "span_in_future_total",
```

Comment:
the suffix on prometheus metrics [should be the unit](https://prometheus.io/docs/practices/naming/). something like?

```
tempo_spans_distance_in_future_seconds
```

wdyt about putting these in the dataquality package:

https://github.com/grafana/tempo/blob/main/pkg/dataquality/warnings.go